.github/workflows/test.yml

```
@@ -141,6 +141,23 @@ jobs:
         run: |
           dpkg -s ros-humble-osrf-testing-tools-cpp
 
+  test_ros2_iron_skip_rosdep_install:
+    name: Build and test current ROS repo
```

Since this name is displayed in the list of CI jobs in every PR, this should be changed to be less generic/example-y:

```suggestion
    name: Test rosdep check/skip install options
```